packages/models/src/agreement/agreementEvents.ts

```diff
@@ -48,4 +48,7 @@ export type AgreementEvent =
       type: "AgreementContractAdded";
       data: AgreementContractAddedV1;
     }
-  | AgreementConsumerDocumentAdded;
+  | {
+      type: "AgreementConsumerDocumentAdded";
+      data: AgreementConsumerDocumentAddedV1;
+    };
```

Comment:
already defined [here](https://github.com/pagopa/interop-be-monorepo/pull/121/files#diff-f9138a0de2b8a601330f641f5308ff78dda2ddc11d65a01118aa8728b47b90b4R35)